### ISSUE TITLE
Recent paper list should probably display authors rather than submitters

### ISSUE THREAD

**Issue by joezuntz** [openjournals/theoj#266]
The current "Recent papers" list displays "Submitted by ..." in its summary.  An author list or first author would be more useful.

**Comment by arfon** [comment]
Agreed. We're working on a better way to handle paper metadata (we currently don't really persist _any_) which would address this.

**Comment by arfon** [comment]
/ cc @marcrohloff @freelanceastro

**Comment by marcrohloff** [comment]
We do persist the authors. We can add it trivially but do we want remove the submittor which also means removing the sort by submittor option?

Background: The issue is that we currently sort by submittor, sorting by authors instead is tricky because Arxiv provided them as 'First Last' (for ex 'Arfon Smith') so sorting wouldn't work as expected. Multiple authors present a further difficulty.

**Comment by arfon** [comment]
> We do persist the authors. We can add it trivially but do we want remove the submittor which also means removing the sort by submittor option?

I think that's probably the right choice. I would vote we just ditch the sorting by author (submittor) and just keep the title & submitted date sorting.

**Comment by marcrohloff** [comment]
I didn't change the sorting but the author is now visible, I also fixed the display issues
